var/spack/repos/builtin/packages/py-clamd/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyClamd(PythonPackage):
+    """clamd is a portable Python module to use the ClamAV anti-virus
+    engine on Windows, Linux, MacOSX and other platforms. It requires
+    a running instance of the clamd daemon."""
```

Comment:
Do we need to add a run-dep on `clamd`?

Reply:
Yes, but I haven't started the `clamd` patch yet.

Reply:
Should we wait on this PR until there is a `clamd` package?

Reply:
Yes, I thinks so.